### PR TITLE
OXT-1185: lvm: Support arguments for different versions

### PR DESCRIPTION
**Pull request by eric-ch**
wipesignatures was introduced on 2.02.105, it should not be passed for
earlier versions.

### CODE REVIEW

**Comment by eric-ch** [comment]
Build [1044](http://openxt-builder.ainfosec.com:8010/builders/openxt/builds/1044)

File: common/stages/Functions/library [hunk 1]
```diff
@@ -342,3 +342,31 @@ get_partition_node()
 
     return 0
 }
+
+#-----------------------------------------------------------
+# Usage: version_ge <v1> <v2>
+# Parse and compare versions formatted as "major.minor.micro".
+# Return 0 if v1 >= v2, else return 1.
+version_ge() {
```
Comment (jean-edouard) [inline common/stages/Functions/library]:
```
sort -V <<EOF | tail -1
$1
$2
EOF
```
:)

Comment (eric-ch) [inline common/stages/Functions/library]: much shorter for sure! But pipes, reads from stdin and would still require processing of the output in the end ;). I'll switch to that if you think it is preferable though.

Comment (jean-edouard) [inline common/stages/Functions/library]: Nah, you make good points... You do call the `[` binary 5 times though! :)

Comment (eric-ch) [inline common/stages/Functions/library]: True but I am not familiar with ash, does it have the ```[[``` built-in? The rest of the script pretty much imply the opposite (only use ```[```).

**Comment by jean-edouard** [comment]
Built here: http://openxt-builder.ainfosec.com:8010/builders/openxt/builds/1046

**Comment by jean-edouard** [comment]
Laptops won't reboot to OpenXT after install.

**Comment by eric-ch** [comment]
This is still quite broken, will re-open later.

**Pull request closed without merging**